.travis.yml

```diff
@@ -19,7 +19,7 @@ before_cache:
 - mkdir -p vendor/assets
 - mv spec/manageiq/vendor/assets/bower_components vendor/assets
 - cp bower.json vendor/assets/bower_components
-after_script: bundle exec codeclimate-test-reporter
+after_script: if [ "$TEST_SUITE" = "spec" ]; then bundle exec codeclimate-test-reporter; fi
```

Comment:
I guess it could be `after_success`, but we've been using `after_script` in every other repo (about a dozen or so so far) without problems.  For consistency, I'd rather leave it this way for now (and also I feel a change like that belongs in a separate PR).

I will put this on my consideration list for the next round of repo updates.